.github/workflows/reusable_compatibility.yml

```diff
@@ -107,9 +107,11 @@ jobs:
 
     - name: Run "tag" UMF tests with latest UMF libs (warnings enabled)
       working-directory: ${{github.workspace}}/tag_version/build
+      # GTEST_FILTER is used below to skip test that is not compatible
       run: >
         UMF_LOG="level:warning;flush:debug;output:stderr;pid:no" 
         LD_LIBRARY_PATH=${{github.workspace}}/latest_version/build/lib/ 
+        GTEST_FILTER="-*umfIpcTest.GetPoolByOpenedHandle*" 
```

Comment:
Could you explain why this test is not compatible?

Reply:
I have already found the answer:
https://github.com/oneapi-src/unified-memory-framework/pull/998/files#r1965460346
:-)